e2e/selectors.ts

```diff
@@ -46,9 +46,9 @@ export const createSelectors = ({ screen, page, within }: Ctx) => {
       },
     },
     panels: {
-      left: () => screen.getByTestId("panel-left"),
+      left: () => screen.getByTestId("panel-body-L"),
```

Comment:
Wondering if we could maybe extract this to a const and reuse it here and in the layout file, so we **can't** forget about updating it in the future? Maybe it could be adapted for all test-ids across the app, what do you think? Or should the "testId" name automatically ring a bell to update an e2e test? 😅

Reply:
I think the broken test should automatically ring a bell 😁 I will try to make a const file 👍